tests/testDoubleGaussianPsf.py

```diff
@@ -34,14 +34,8 @@
 import lsst.afw.display.utils as displayUtils
 import lsst.meas.algorithms as measAlg
 import lsst.pex.exceptions as pexExceptions
-import lsst.pex.logging as logging
 import lsst.utils.tests
 
-try:
-    type(verbose)
-except NameError:
-    verbose = 0
-    logging.Trace_setVerbosity("algorithms.Interp", verbose)
```

Comment:
I think it'd be good to do the same thing here that you do in testPsfDetermination: set logging to INFO or DEBUG or TRACE with comment on how to increase log level.